### ISSUE TITLE
Add "Move cursor to original location" to the Marker tab

### ISSUE THREAD

**Issue by HeyHo2roar** [Friendly0Fire/GW2Radial#344]
All other tabs in the menu have the option to enable "Move cursor to original location after release". The "Marker tab appears to be missing that.

Are there some limitations in that category especially I may have overlooked?
Will the marker be placed at the spot you let go of the bound key or at the centre of the wheel?

Personally it feels pretty unusable without this option enabled, since I rely on the movements of my Hand to know where my cursor is.

Ty for considering this.

**Comment by Friendly0Fire** [comment]
Yes, the reason this one doesn't have that option is that the cursor's location is important to only this radial menu, so I have to make concessions to keep it functional, otherwise you wouldn't be able to choose where to place the marker.
